src/components/playback/playbackmanager.js

```diff
@@ -2502,6 +2492,33 @@ class PlaybackManager {
             }
         }
 
+        function detectBitrate(apiClient, item, mediaType, playOptions) {
+            // FIXME: This is gnarly, but don't want to change too much here in a bugfix
+            return Promise.resolve()
+                .then(() => {
+                    if (playOptions.fullscreen) {
+                        loading.show();
+                    }
```

Comment:
Can we move this to `then` in between:
https://github.com/jellyfin/jellyfin-web/blob/70b9aa46114b3d60435499e33a85b93c162ecaa9/src/components/playback/playbackmanager.js#L2287-L2288
so that `loading.show/hide` are not split over the code?